app/services/state_file/state_information_service.rb

```diff
@@ -1,16 +1,37 @@
 module StateFile
   class StateInformationService
     class << self
+      [
+        :download_form_name,
+        :intake_class,
+        :mail_voucher_address,
+        :pay_mail_online_link,
+        :refund_url,
+        :state_name,
+        :survey_link,
+        :tax_payment_url,
+        :vita_link,
+        :voucher_path,
+      ].each do |attribute|
+        define_method(attribute) do |state_code|
+          raise StandardError, "No state code '#{state_code}'" if !active_state_codes.include?(state_code)
+
+          value = STATES_INFO[state_code.to_sym][attribute]
+          raise StandardError, "State '#{state_code}' does not have '#{attribute}'" if !value
+          value
+        end
+      end
+
       def active_state_codes
         STATES_INFO.keys.map(&:to_s)
       end
 
-      def state_name(state_code)
-        unless active_state_codes.include?(state_code)
-          raise StandardError, state_code
-        end
-
-        STATES_INFO[state_code.to_sym][:name]
+      def state_intake_classes
+        STATES_INFO.map { |_, attrs| attrs[:intake_class] }
+      end
+      
+      def state_intake_class_names
+        state_intake_classes.map(&:to_s).freeze
       end
```

Comment:
was on the fence about this because it's so similar to `state_intake_classes` but i didn't like having to `.map(&:to_s).freeze` in all of the places where it needs them as strings (there are kind of a lot). if you have other ideas for this tho lmk